docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,645 @@
+= Migration Tutorial: From APPUiO Public to APPUiO Cloud
+
+This document provides an overview of all the tasks required to move your applications from APPUiO Cloud to APPUiO Public.
+
+== Main Differences
+
+Before starting the migration of your applications from APPUiO Public to APPUiO Cloud, please check the xref:differences-to-public.adoc[document explaining all the major differences] between both platforms.
+
+=== Timezone
```

Comment:
I don't think that the Timezone is the most important difference to especially highlight here. I'd rather improve the `differences-to-public.adoc` regarding that topic.

Reply:
Removed